tests/test_conditions.py

```diff
@@ -0,0 +1,126 @@
+import ast
+import pytest
+import sys
+
+from vulture import utils
+from . import check_unreachable
+from . import v
+assert v  # Silence pyflakes
+
+
+def check_condition(code, result):
+    condition = ast.parse(code, mode='eval')
+    assert utils.evaluate_condition(condition) == result
+
+
+def test_false():
+    check_condition('False', False)
+    check_condition('None', False)
+    check_condition("0", False)
+    check_condition("0", False)
+    if sys.version_info > (3, 0):
+        check_condition("1 - 1", False)
+
+
+def test_empty():
+    check_condition("''", False)
+    check_condition("[]", False)
+    check_condition("{}", False)
+
+
+def test_true():
+    check_condition("True", True)
+    check_condition("2", True)
+    check_condition("['foo', 'bar']", True)
+    check_condition("{'a': 1, 'b': 2}", True)
+
+
+def test_errors():
+    conditions = ['foo',
+                  'foo and False',
+                  '__name__ == "__main__"',
+                  'False and 1',
+                  'False and foo',
+                  'chr(-1)',
+                  'getattr(True, "foo")',
+                  'hasattr(str, "foo")',
+                  'isinstance(True, True)',
+                  'not False']
+    for condition in conditions:
+        with pytest.raises(ValueError) as e:
+            check_condition(condition, False)
+        assert e.match("Condition cannot be evaluated")
```

Comment:
I preferred it because [The official pytest documentation](https://docs.pytest.org/en/latest/assert.html) uses it.
Should we?